packages/core/admin/admin/src/components/LeftMenu.tsx

```diff
@@ -134,19 +134,29 @@ const LeftMenu = ({ generalSectionLinks, pluginsSectionLinks }: LeftMenuProps) =
             })}
           >
             {pluginsSectionLinks.map((link) => {
-              const Icon = link.icon;
+              const LinkIcon = link.icon;
+              const { lockIcon = false } = link;
```

Comment:
Why on deconstruct this? if it doesn't exist it will still be falsey if that's what you're concerned about.